src/pds/registry/utils/geostac/create_lola_pds4.py

```diff
@@ -0,0 +1,166 @@
+"""Script to scrape GeoSTAC for Lola point clouds and make pds4 xml."""
+import argparse
+import importlib
+from datetime import date
+from pathlib import Path
+
+import requests
+from jinja2 import Environment
+from pds.registry.utils.geostac import templates
+
+
+def create_product_external(item):
+    """Creates Product_External for given item.
+
+    :param item: item to fill out pds4 xml information with
+
+    :return: pds4 xml
+    """
+    # create env
+    env = Environment()
+
+    with importlib.resources.open_text(templates, "product-external-template.xml") as io:
```

Comment:
One risk using text templating like Jinja2 with XML files is that the substituted text could contain markup or characters that are special to XML.

Is there any chance `lid`, `title`, `bb_west`, `file_name`, etc. might contain characters like `<`, `&`, etc.?

If not, then looks good to go!

Reply:
I think it's fine for lola since these datasets were built with data derived from data that's in the pds. I didn't run into issues with the api given to me to scrape these layers, and I'd be surprised if there are issues given that it wouldn't have gotten "bad/ invalid" characters in the pds data it used.

Reply:
Good to know!